doc/refactorings.md

```diff
@@ -893,6 +894,55 @@ class Hello
 }
 ```
 
+Expand Class
+------------
+
+Instead of include `use` statement for a class under cursor,
+expand the class name from unqualified name to fully qualified name.
+
+- **Command**: _VIM function only_
+- **VIM plugin**: _VIM function only_
+- **VIM function**:`:call phpactor#ClassExpand()`
+
+### Motivation
+
+Although `use` statement make code cleaner, but sometime the code would be more
+readable if we specify the full qualified name right in the place. For example,
+we might register a list of listener in a centralized file. We would be more
+pleasant to know the exact class right in the place.
```

Comment:
- sometime => sometimes
- listener => listeners
- could remove "right in the place" and "centeralized"?
- could remove "We would be more pleasant to know the ..." sentence?